angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts

```diff
@@ -268,6 +268,26 @@ export const defaultStory = () => ({
           Sed ut perspiciatis unde omnis iste natus error sit <a href="#nogo" sprkLink class="sprk-b-Link--inline-light"> inline link</a> accusantium doloremque laudantium
         </p>
       </sprk-stack>
+      <div appStoreLinksSlot>
```

Comment:
Was there a reason you didn't put the appStoreLinkSlot on the content that you are passing in? Or did we need an empty div here?

Reply:
you're right it's unnecessary, moved it to the real container.